cli/config/compose/services/elasticsearch/docker-compose.yml

```diff
@@ -6,10 +6,9 @@ services:
       - discovery.type=single-node
       - ES_JAVA_OPTS=-Xms512m -Xmx512m
       - xpack.security=false
-      - xpack.monitoring.collection.enabled=true
+      - xpack.monitoring.collection.enabled=${xpackMonitoringCollection:-true}
       - ELASTIC_USERNAME=elastic
       - ELASTIC_PASSWORD=p4ssw0rd
     image: "docker.elastic.co/elasticsearch/elasticsearch:${elasticsearchTag}"
     ports:
-      - "9200:9200"
-      - "9300:9300"
+      - "${elasticsearchPort:-9200}:9200"
```

Comment:
I can't comment on line 3 (because it's not part of the diff) but WDYT about calling this service `productionES`, to mirror the other Elasticsearch service, `monitoringES`?

Reply:
BTW, "production ES cluster" and "monitoring ES cluster" are terms that are commonly used in the context of Stack Monitoring.

Reply:
I'm afraid that the service name is not configurable in runtime, so the `elasticsearch` name is hardcoded.

Reply:
Gotcha, so just so I'm understanding this right: `elasticsearch` here refers to the "production" ES cluster, right?

Reply:
Yes: anything under `services` is reusable to be aggregated to a running compose on the fly, that's why they are generic

Reply:
On the other hand, I'm seeing a potential problem if somebody wants to add two services of the same kind in an already running compose: service names would collide 🤔